client/js/webpush.ts

```diff
@@ -28,7 +26,7 @@ socket.once("push:issubscribed", function (hasSubscriptionOnServer) {
 				// If client has push registration but the server knows nothing about it,
 				// this subscription is broken and client has to register again
 				if (subscription && hasSubscriptionOnServer === false) {
-					subscription.unsubscribe().then((successful) => {
+					void subscription.unsubscribe().then((successful) => {
```

Comment:
what's the void for?

Reply:
eslint rule. If we aren't using async/await we either need to handle all promises with catch or explcitly mark them with void.

Reply:
Add a catch then? You did in other places.

Reply:
Replaced a few void calls, left in the ones that should be replaced with async/await (like when we move to https://pinia.vuejs.org/, the vuex 5 rebrand)